src/cmd-koji-upload

```diff
@@ -406,6 +406,32 @@ class _KojiBase():
         return session
 
 
+class Search(_KojiBase):
+    """
+    Search for builds
+    """
+
+    def __init__(self, profile):
+        """
+        Creates a new instance for search.
+
+        :param profile: Koji profile name in /etc/koji.conf.d
+        :type str
+        """
+        super().__init__(profile)
+
+    def get_state(self, nvr):
+        """
+         Return the build state.
+        :param nvr: The nvr name from Brew
+        """
+
+        info = self.session.getBuild(nvr, strict=False)
+        if (info):
+            return (info['state'])
+        return 0
```

Comment:
Since I'm printing the return [here](https://github.com/coreos/coreos-assembler/pull/3519/commits/f184228b8126166d9665a30305672491824a6594), it will be printed only if info exits. The 'return 0' will be returned as empty. Nothing will show. At least in the tests I did, it was what I got.

Reply:
The states:
https://pagure.io/koji/blob/master/f/tests/test_cli/test_import.py#_73-_78
```
  | # koji.BUILD_STATES
-- | --
  | self.bstate = {
  | 'BUILDING': 0,
  | 'COMPLETE': 1,
  | 'DELETED': 2,
  | 'FAILED': 3,
  | 'CANCELED': 4,
  | }

```

Reply:
> Maybe we should change it to other number? The '0' state means BUILDING, if a job failed/ was cancel, it means Brew is in BUILDING state because it is probably waiting for a timeout. And eventually it will be garbage collected.

Yes, I agree.  if `if (info)` evaluates to false is that an error? we probably shouldn't be returning a valid build state in that case?

Reply:
Not an error  I would say. It means there isn't any build with this NVR yet. How about '-1' to we won't hit any case?